.env.local.example

```diff
@@ -1,4 +1,8 @@
 # create local copy of this file and rename to .env.local
 MONGO_URI={mongo-uri-here}
 NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={clerk-pub-key-here}
-CLERK_SECRET_KEY={clerk-secret-key-here}
\ No newline at end of file
+CLERK_SECRET_KEY={clerk-secret-key-here}
+NEXT_PUBLIC_CLERK_SIGN_IN_URL=/sign-in
+NEXT_PUBLIC_CLERK_SIGN_UP_URL=/sign-up
+NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/
+NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/
```

Comment:
these vars are fine to put in the env example since they don't contain any sensitive keys, but we typically want to put a placeholder here like the vars above so we don't expose them publicly